.github/workflows/test.yml

```
@@ -1268,7 +1268,13 @@ jobs:
 
         # workspaces are added here as their lint issues are fixed
       - name: Lint check
-        run: turbo run lint --filter=create-turbo --filter=turbo-ignore --filter=@turbo/types --filter=eslint-config-turbo --filter=eslint-plugin-turbo
+        run: turbo run lint \
+          --filter=create-turbo \
+          --filter=eslint-config-turbo \
+          --filter=eslint-plugin-turbo \
+          --filter=turbo-ignore \
+          --filter=@turbo-test-utils
```

Double checked and we do use `/` in the `package.json` name.
```suggestion
          --filter=@turbo/test-utils
```